presentation/src/main/java/com/android254/presentation/common/components/OrganizedBySection.kt

```diff
@@ -49,7 +53,7 @@ fun OrganizedBySection(
         modifier = modifier
             .fillMaxWidth()
             .background(
-                color = ChaiLightGrey,
+                color = MaterialTheme.colorScheme.surface,
```

Comment:
@Borwe why are you moving away from colors of our design system

Reply:
I thought I was supposed to move to MaterialTheme.colorScheme.X so that dark and light mode we wouldn't have to use if statements to check according to this request:

https://github.com/droidconKE/droidconKE2023Android/pull/140#discussion_r1230950559